.github/workflows/publish-release-type.yaml

```diff
@@ -66,3 +66,21 @@ jobs:
           channel: team-pitch-black
           color: red
           verbose: true
+      - name: slack on success - aws-dev
+        if: ${{ success() }}
+        uses: actions-ecosystem/action-slack-notifier@fc778468d09c43a6f4d1b8cccaca59766656996a
+        with:
+          slack_token: ${{ secrets.slackToken }}
+          message: ":cool-doge: ${{ inputs.name }} successful"
```

Comment:
Should the message be different? @Himangini ?

Reply:
It'll be cool if we can link to the release like `https://github.com/weaveworks/eksctl/releases/tag/v0.92.0-rc.0` for example but that's totally optional and up to you 💡

Reply:
I don't think I can format the link nicely. There is no shortcut in text in slack like in Markdown. I can add the link but it would look really ugly. :)

Reply:
`[your text](the link)` this doesn't work? 🤔  anyway it was a nice thing to have, not an issue if it's not there :D

Reply:
apparently, you can do something like this `<http://www.example.com|This message *is* a link>`